Tools/cases_generator/opcode_metadata_generator.py

```diff
@@ -330,6 +330,32 @@ def generate_pseudo_targets(analysis: Analysis, out: CWriter) -> None:
     out.emit("}\n\n")
 
 
+def generate_tier2_properties(analysis: Analysis, out: CWriter) -> None:
+    table_size = len(analysis.instructions) + len(analysis.uops)
+
+    out.emit(f"extern const uint8_t _PyOpcode_ispure[{table_size}];\n")
```

Comment:
This doesn't need its own table. Just set a bit in the `_PyUop_Flags` and `_PyOpcode_opcode_metadata` tables